src/clan_arena.c

```diff
@@ -744,6 +606,116 @@ void CA_show_greeting(gedict_t *self)
 	}
 }
 
+void CA_UpdateClients(void)
+{
+	gedict_t *p;
+
+	static double lastupdate = 0;
+
+	if (g_globalvars.time - lastupdate < 0.5)
```

Comment:
the code format of the if() statement is with brackets:
if ()
{

}

even if there is only one single statement